README.md

```diff
@@ -281,8 +206,6 @@ I would like to thank...
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+This project is [MIT](./LICENSE.md) licensed.
 
```

Comment:
- [OPTIONAL] The link to your MIT is broken. Kindly add a valid LICENSE. You can follow the instructions on the right side of the article attached below to set up a valid _**LICENSE**_
https://choosealicense.com/licenses/mit/